lib/aruba/processes/spawn_process.rb

```diff
@@ -83,7 +83,7 @@ def start
             sleep startup_wait_time
           end
         rescue ChildProcess::LaunchError => e
-          raise LaunchError, "It tried to start #{cmd}. " + e.message
+          raise LaunchError, "It tried to start #{commandline}. " + e.message
```

Comment:
Oh, and after this fix, the affected tests on AppVeyor actually show the real errors now (instead of triggering the missing variable error).